tests/test_configuration.py

```diff
@@ -60,7 +65,12 @@
     'SOME_BOOL3': True,
     'SOME_BOOL4': False,
     'SOME_BOOL5': False,
-    'SOME_BOOL6': False
+    'SOME_BOOL6': False,
+    'SOME_LIST1': [3, 15, -999],
+    'SOME_LIST2': [0.2, 3.5, -9999.],
+    'SOME_LIST3': [datetime(2022, 12, 25, 0, 0, 0), datetime(2022, 12, 25, 18, 45, 0)],
+    'SOME_LIST4': [True, False, True],
+    'SOME_LIST5': [0.2, 15, datetime(2022, 12, 25, 0, 0, 0), False],
```

Comment:
It may also be worth it to add a test for a string among numbers/datetimes, e.g.
```suggestion
    'SOME_LIST5': [0.2, 15, datetime(2022, 12, 25, 0, 0, 0), False],
    'SOME_LIST6': [0.2, 'foo', 15, datetime(2022, 12, 25, 0, 0, 0), False],
```
This would also need to be added to `file0` and `list_dtypes`.

Reply:
Added a plain string to the list test.